meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-bsp/u-boot/u-boot-ti-staging-mfgtool_%.bbappend

```diff
@@ -0,0 +1,3 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/u-boot-ti-staging:"
```

Comment:
Do we need this bbappend? The staging-mfgtool recipe is ours, so why not add this as part of the recipe itself?

Reply:
fixed

Reply:
I think the sugestion here is to move everything from this bbappend file to the main recipe so it can be droped

Reply:
Yeah

Reply:
got it finally :)